scripts/network-launch/launch.sh

```diff
@@ -15,8 +17,10 @@ generate_chain_spec () {
 }
 
 start_boot () {
+  ip=${bootNodeIP}
+  clone_scripts_if_necessary ip
```

Comment:
Do you want to use `bootNodeIP` everywhere here? Otherwise it looks strange

Reply:
Done but let me know if you understood you correctly.
Also within the start_node function we have altering IP & Host references

Reply:
You've introduced `ip` variable that references `bootNodeIP`. At the same time, `bootNodeIP` is using already in the method. We should select one approach and use it. 
Please, revert changes you've done for the `*host`.

Reply:
Done